torch/_tensor.py

```diff
@@ -1012,6 +1012,34 @@ def __torch_function__(cls, func, types, args=(), kwargs=None):
             ret = func(*args, **kwargs)
             return _convert(ret, cls)
 
+    def __dlpack__(self, stream=None):
+        """
+        Creates a dlpack capsule of the current tensor to
+        be exported to other libraries.
+
+        This function will be called from the `from_dlpack` method
+        of the consumer library.
+
+        Args:
+            stream (integer or None): Object that represents a stream and provides
+                a `synchronize` method. Optional.
```

Comment:
Oh wait, given the below I think you are extending what's in the array API standard on purpose? What is the use case here? Presumably the caller is coming from another library, would they create/own a PyTorch `Stream` object and work with that instead of with their own stream?

Reply:
I didn't understand the situation correctly :), I didn't check the actual array API documentation but only the data exchange section which didn't explicitly specify what the stream was, so I went forward with this assumption for a first try.
Will fix it now that I got it! thanks!